pkg/metrics/metrics_test.go

```diff
@@ -130,25 +129,6 @@ func TestEventProcessingStatsReporter(t *testing.T) {
 	})
 }
 
-// OpenCensus metrics carry global state that need to be reset between unit tests.
-func resetMetrics(t *testing.T) {
-	t.Helper()
-
-	metricstest.Unregister(
-		"event_processing_success_count",
-		"event_processing_error_count",
-		"event_processing_latencies",
-	)
-
-	MustRegisterEventProcessingStatsView()
-
-	metricstest.AssertNoMetric(t,
-		"event_processing_success_count",
-		"event_processing_error_count",
-		"event_processing_latencies",
-	)
-}
```

Comment:
This helper is useful for potentially any adapter test, so I created a global test helper for it.